### PR TITLE
US859:  Updates from Call Note Design Meeting

### PR DESCRIPTION
**Pull request by aujosh84**
1.  Added "Difficulty with lactation" to breastfeeding tab.
2.  Added "End" button to navigational buttons

### CODE REVIEW

File: Dashboard/va.gov.artemis.ui.data/Models/PatientContact/BreastfeedingCallTab.cs [hunk 1]
```diff
@@ -248,6 +256,9 @@ public override string GetNoteText()
                 if (this.BreastfeedingAssessment)
                     sb.AppendLine("Assessed for difficulties with breastfeeding, answered questions, referred to pediatrician as needed");
 
+                if (this.BreastfeedingLactation)
+                    sb.AppendLine("Patient is having difficulties with lactation");
```
Comment (aujosh84) [inline Dashboard/va.gov.artemis.ui.data/Models/PatientContact/BreastfeedingCallTab.cs]: I am not a fan of the formatting here.  I really want to add braces, but I kept it formatted the same as the rest of the code.  If we were making any other major modifications to the baseline I would vote for changing everything back to proper standards, but for now my vote is to leave it the way we found it.

File: Dashboard/va.gov.artemis.ui.data/Models/PatientContact/CallTabBase.cs [hunk 2]
```diff
@@ -39,6 +39,9 @@ public abstract class CallTabBase: PatientRelatedModel
         //public int NextTabIndex { get { return ((this.TabIndex + 1) < this.TabCount) ? this.TabIndex + 1 : -1; } }
         public int NextTabIndex { get { return this.TabIndex + 1; } }
 
+        // *** Return the index of the "last" tab ***
+        public int LastTabIndex { get { return TabCount - 1; } }
```
Comment (alinafrey) [inline Dashboard/va.gov.artemis.ui.data/Models/PatientContact/CallTabBase.cs]: this.TabCount?

Comment (aujosh84) [reply Dashboard/va.gov.artemis.ui.data/Models/PatientContact/CallTabBase.cs]: good catch.

File: Dashboard/va.gov.artemis.ui.data/Models/PatientContact/BreastfeedingCallTab.cs (hunk 1 above)
Comment (alinafrey) [reply Dashboard/va.gov.artemis.ui.data/Models/PatientContact/BreastfeedingCallTab.cs]: I agree. I'm the same way. I don't like the style with the missing braces. One wrong indentation can mess up the way you may read things.

Comment (alinafrey) [reply Dashboard/va.gov.artemis.ui.data/Models/PatientContact/BreastfeedingCallTab.cs]: But I agree, we should continue with the same style, until we decide to pull the trigger and change it for the whole code base.

File: Dashboard/va.gov.artemis.ui/Views/PatientContact/_CallTab.cshtml [hunk 3]
```diff
@@ -86,6 +86,7 @@
                 @if (!Model.IsLast)
                 {
                     <button type="button" id="next-btn" class="btn btn-primary btn-submit" data-nav="@Model.NextTabIndex">Next&nbsp;&nbsp;<span class="glyphicon glyphicon-arrow-right"></span></button>
+                    <button type="button" id="next-btn" class="btn btn-primary btn-submit" data-nav="@Model.LastTabIndex"><span class="glyphicon glyphicon-ok"></span>&nbsp;&nbsp;End</button>
```
Comment (alinafrey) [inline Dashboard/va.gov.artemis.ui/Views/PatientContact/_CallTab.cshtml]: Suggest giving the button a different id, something like  id="end-btn"

Comment (aujosh84) [reply Dashboard/va.gov.artemis.ui/Views/PatientContact/_CallTab.cshtml]: agreed.  wish this had a like button.

**Review by alinafrey — APPROVED** [review]

**Pull request merged**